com.unity.render-pipelines.high-definition/Editor/VFXGraph/Outputs/VFXLitSphereOutput.cs

```diff
@@ -57,11 +57,23 @@ protected override IEnumerable<string> filteredOutSettings
                 foreach (var setting in base.filteredOutSettings)
                     yield return setting;
 
-                yield return "cullMode";
-                yield return "blendMode";
-                yield return "useAlphaClipping";
-                yield return "doubleSided";
-                yield return "shaderGraph";
+                yield return nameof(cullMode);
+                yield return nameof(blendMode);
+                yield return nameof(useAlphaClipping);
+                yield return nameof(doubleSided);
+                yield return nameof(shaderGraph);
+            }
+        }
+
+        protected override IEnumerable<string> untransferableSettings
+        {
+            get
+            {
+                foreach (var setting in base.untransferableSettings)
+                {
+                    yield return setting;
+                }
+                yield return nameof(shaderGraph);
```

Comment:
Ok ⏫ We discussed about this, the "Convert Output" was behaving weirdly, the untransferableSettings prevent the creation of Lit or Cube Sphere output 🟢